lib/errors.js

```diff
@@ -264,7 +242,8 @@ exports.ConnectionRefusedError = ConnectionRefusedError;
 /**
  * Thrown when a connection to a database is refused due to insufficient privileges
  * @extends ConnectionError
- * @constructor
+ * @memberof Errors
+ * @memberof Sequelize
```

Comment:
Yeah, I still need to figure out how we document how to access the errors - I think it makes sense to have them in a separate namespace, but on the other hand I still want to document that you can access them as  `sequelize.HostNotFoundError` etc.